openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -443,6 +443,18 @@ jQuery(function () {
         $('details[open]').not(this).removeAttr('open');
     });
 
+    $('.header-dropdown').on('keydown', function (event) {
+        if (event.key === 'Escape') {
+            $('details[open]').not(this).removeAttr('open');
```

Comment:
`<details>` elements are used for a few miscellaneous things throughout the site; we don't need them all to close just the header ones.

```suggestion
            $('.header-dropdown > details[open]').not(this).removeAttr('open');
```